integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/DataManagementFunc.scala

```diff
@@ -49,105 +44,6 @@ object DataManagementFunc {
 
   private val LOGGER = LogServiceFactory.getLogService(this.getClass.getCanonicalName)
 
-  def deleteLoadByDate(
```

Comment:
Just confirm  :   this method is for spark 1.6?

Reply:
No one use this function